### DISCUSSION TITLE
This Serverless Function has timed out.

### DISCUSSION THREAD

**Discussion by saidhasyim** [vercel/community#2335] [Help]
### Summary

Hi,

Sorry, I’ve spent days trying to figure out what’s wrong with the serverless function suddenly not working. It was working fine about 4 days ago. I only made minor adjustment to my codes since then and it shouldn‘t cause this. Even after I rollback, it still does not work. Would you shed some light where I can investigate further?

Note that it is working fine in my localhost.

I created a test function with just the below content:
`module.exports = (req, res) => {
    res.send("Hello, World!")
}`

This is what the log says:
![image](https://user-images.githubusercontent.com/111839161/236623835-30902ff1-5f73-4a71-af45-0412bf2f213a.png)

Would you help me please? This was working fine 4 days ago.

I have googled all similar issues and tried what i saw from the suggestions. I have also checked this: https://vercel.com/guides/what-can-i-do-about-vercel-serverless-functions-timing-out

**Comment by swarnava** [comment]
Hi,

Do you have a reproducible repository which we can look at?

**Reply by saidhasyim** [reply]
I will prepare a lightweight reproducible version now.

**Reply by saidhasyim** [reply]
Hi Swarnava, I can no longer replicate it. It somehow worked after I selected "purge cache" during deployment. Thanks for getting back to me.

**Comment by saidhasyim** [comment]
not longer an issue

**Reply by Lshiroc** [reply]
So how did you solve it?

**Reply by varkelth** [reply]
BRO TELL US

**Reply by saidhasyim** [reply]
Hi lshiroc and Sutoreitsoh, my apologies for responding late. It worked after I selected "purge cache" during deployment. Since then, I never encounter this error.
